x-pack/plugins/apm/public/components/shared/charts/Timeline/__test__/Timeline.test.js

```diff
@@ -44,4 +44,27 @@ describe('Timeline', () => {
 
     expect(toJson(wrapper)).toMatchSnapshot();
   });
+
+  it('should not crash if traceRootDuration is 0', () => {
+    const props = {
+      traceRootDuration: 0,
+      width: 1000,
+      duration: 0,
+      height: 116,
+      margins: {
+        top: 100,
+        left: 50,
+        right: 50,
+        bottom: 0
+      }
+    };
+
+    const wrapper = mount(
+      <StickyContainer>
+        <Timeline {...props} />
+      </StickyContainer>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
Thanks for adding a test here. Maybe instead of a snapshot for this one, we can do:
```js
const mountTimeline = () =>
  mount(
    <StickyContainer>
      <Timeline {...props} />
    </StickyContainer>
  );

expect(mountTimeline).not.toThrow();
```

which produces this output if it throws:
```
  ● Timeline › should not crash if traceRootDuration is 0

    expect(function).not.toThrow()

    Expected the function not to throw an error.
    Instead, it threw:
      TypeError: Cannot read property 'prototype' of undefined
```

Reply:
I agree. `not.toThrow();` is better 👍